monitoring/local_monitoring/kafka_demo/kafka-demo.sh

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash
+#
+# Copyright (c) 2025 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Default values
+CLUSTER_TYPE="ephemeral"
+KRUIZE_IMAGE="quay.io/khansaad/autotune_operator"
+KRUIZE_IMAGE_TAG="kafka"
+DATASOURCE_URL="http://thanos-query-frontend-example-query-thanos-operator-system.apps.kruize-scalelab.h0b5.p1.openshiftapps.com/"
+DATASOURCE_NAME="thanos-ee-2"
+CLOWDAPP_FILE="./ros-ocp-backend/kruize-clowdapp.yaml"
+BONFIRE_CONFIG="$HOME/.config/bonfire/config.yaml"
+current_dir="$(dirname "$0")"
+LOG_FILE="${current_dir}/kafka-demo.log"
+
+start_demo=1
+skip_setup=0
+repo_name=ros-ocp-backend
+common_dir="${current_dir}/../../../common/"
+
+set -euo pipefail  # Enable strict error handling
+
+source ${common_dir}/common_helper.sh
+source ${current_dir}/../common.sh
+
+function usage() {
+	echo "Usage: $0 [-s|-t] [-i kruize-image] [-u datasource-url] [-d datasource-name]"
+	echo "s = start (default), t = terminate"
+	echo "i = Kruize image (default: $KRUIZE_IMAGE)"
+	echo "c = Cluster type (default: ephemeral)"
+	echo "u = Prometheus/Thanos datasource URL (default: $DATASOURCE_URL)"
+	echo "d = Name of the datasource (default: $DATASOURCE_NAME)"
+	exit 1
+}
+
+# Function to handle errors
+error_exit() {
+    echo "❌ Error: $1"
+    exit 1
+}
+
+# Check if bonfire is installed
+function check_bonfire() {
+if ! command -v bonfire &>/dev/null; then
+    error_exit "Bonfire tool is not installed. Please install it before running this script."
```

Comment:
@khansaad This should be a pre-requisite in the document. Please provide link to the bonfire setup